src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -21,54 +23,97 @@ type TableRowData = RowData & {
 
 // Columns
 
+function cellDefaultTooltip(displayValue: string) {
+  return <Tooltip title={displayValue}>{displayValue}</Tooltip>;
+}
+
+function cellNullTooltip() {
+  return (
+    <Tooltip title="NULL">
+      <i>NULL</i>
+    </Tooltip>
+  );
+}
+
+function cellNumericGeneralizedTooltip(binSize: number, v: number) {
+  const displayValue = v.toString();
+  const tooltip = `[${v}; ${v + binSize})`;
+  return <Tooltip title={tooltip}>{displayValue}</Tooltip>;
+}
+
+function cellTooltip(bucketColumn: BucketColumn | undefined, v: number | boolean | string) {
+  if (
+    bucketColumn &&
+    (bucketColumn.type === 'integer' || bucketColumn.type === 'real') &&
+    bucketColumn.generalization &&
+    bucketColumn.generalization.binSize
```

Comment:
This condition seems overly defensive. When can the `bucketColumn` be `undefined` or `binSize` field be missing?

Reply:
hm, I think I inherited from the code in the other module, where it was indeed nullable, good catch!